core/src/main/scala/org/apache/spark/storage/MemoryStore.scala

```diff
@@ -661,11 +613,11 @@ private[spark] class MemoryStore(blockManager: BlockManager, memoryManager: Memo
   }
 
   /**
-   * Log a warning for when a over size block is detected.
-   *
-   * @param blockId ID of the block we are trying to unroll.
-   * @param finalVectorSize Final size of the vector when the block size limit is reached.
-   */
+    * Log a warning for when a over size block is detected.
+    *
+    * @param blockId ID of the block we are trying to unroll.
+    * @param finalVectorSize Final size of the vector when the block size limit is reached.
+    */
```

Comment:
This is incorrect, JavaDoc-style indentation. ScalaDoc style aligns with the first `*`, not the second.